products/opensuse/main.pm

```diff
@@ -461,6 +461,11 @@ elsif (get_var('SECURITYTEST')) {
 elsif (get_var('SYSTEMD_TESTSUITE')) {
     load_systemd_patches_tests;
 }
+elsif (get_var('DOCKER_IMAGE_TEST')) {
+    boot_hdd_image;
+    load_docker_tests;
```

Comment:
Don't we already schedule the docker tests in another case? I don't think we should schedule them multiple times

Reply:
This needs to be run in a separate test run to separate image failures from host failures.

The TW docker image will only be released if this particular test passed.

Reply:
I agree with @okurz, this will create duplication. It makes sense to have this test separate, but then

1. I will suggest to remove the module from extra tests and rely on this one about docker.
2. Or, in case that the tumbleweed docker image simply needs docker to be installed, then we should split the module **docker** into **install_docker** and **docker**. Then your test module as **tumbleweed_docker_image**.

What do you think?

Reply:
I don't agree - IMO we need both tests. `load_docker_tests` is not a prerequisite, but just nice to have as a quick indication whether it's really the image or the host which is broken. If the host is broken, there's no point to test the image either.